tools/lxdclient/client_network.go

```diff
@@ -103,12 +101,12 @@ func CreateDefaultBridgeInDefaultProfile(client creator) error {
 
 	config, _, err := client.GetProfile("default")
 	if err != nil {
-		return err
+		return errors.Trace(err)
 	}
 	_, ok := config.Devices["eth0"]
 	if ok {
 		/* don't configure an eth0 if it already exists */
-		return nil
+		return errors.Trace(err)
```

Comment:
We know that err will be nil here, so it doesn't matter, but I would be inclined to leave it as a nil return.

Reply:
yeah, just overzealous copy & paste